src/map/script.c

```diff
@@ -23031,6 +23031,58 @@ BUILDIN(navigateto)
 }
 
 /**
+ * adopt("<parent_name>","<baby_name>");
+ * adopt(<parent_id>,<baby_id>);
+ */
+BUILDIN(adopt)
+{
+	struct map_session_data *sd, *b_sd;
+	enum adopt_responses response;
+
+	if (script_isstringtype(st, 2)) {
+		sd = script->nick2sd(st, script_getstr(st, 2));
+		if (sd == NULL) {
+			ShowError("buildin_adopt: Non-existant parent character %s requested.\n", script_getstr(st, 2));
```

Comment:
Note that `script->nick2sd()` already prints an extensive error if it can't find the character. This error message is redundant